src/app/screens/Onboard/TestConnection/index.tsx

```diff
@@ -47,13 +47,14 @@ export default function TestConnection() {
       setErrorMessage(message);
     } finally {
       setLoading(false);
+      setErrorMessage("");
```

Comment:
why that? finally runs always (no matter if it is successful or an error happens). So does this not overwrite the `setErrorMessage` in the error case in line 47?

Reply:
True, I just noticed that if the timeout error is showing and a success is happening during that, error and success show at the same time. Adjusted it.